python/llm/src/bigdl/llm/transformers/low_bit_linear.py

```diff
@@ -291,7 +291,7 @@ class MatMulLowBit(torch.autograd.Function):
     def forward(ctx, A, weight):
         ctx.is_empty = False
         import linear_q4_0
-        result = linear_q4_0.forward_new(A, weight.data, weight.qtype)
+        result = linear_q4_0.forward_new(A, weight.data, weight.qtype, True)
```

Comment:
I am a little confused. I thought `forward_new` only requires 3 arguments?

Reply:
Sorry, I forgot to upload the code in llm.cpp. The related PR: https://github.com/intel-analytics/llm.cpp/pull/111